resources/css/app.css

```diff
@@ -1,3 +1,5 @@
-@import 'tailwindcss/base';
-@import 'tailwindcss/components';
-@import 'tailwindcss/utilities';
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```

Comment:
you have added "

Reply:
didnt get

Reply:
compare the red and green lines @mammubarak

Reply:
Ok ok i will put "